types/react/index.d.ts

```diff
@@ -2272,7 +2273,7 @@ declare namespace React {
         ascent?: number | string;
         attributeName?: string;
         attributeType?: string;
-        autoReverse?: number | string;
```

Comment:
Any idea if this is supposed to accept `0` or `1`?  I don't see anything in the code you linked, but I assume there was some reason for allowing `number` in the old type.

Reply:
I changed this as well because the w3.org recommendation explicitly lists `"true" | "false"`

https://www.w3.org/TR/REC-smil/smil-timemanip.html#adef-autoReverse